xbmc/guilib/StereoscopicsManager.cpp

```diff
@@ -509,6 +509,8 @@ void CStereoscopicsManager::OnPlaybackStarted(void)
   STEREOSCOPIC_PLAYBACK_MODE playbackMode = (STEREOSCOPIC_PLAYBACK_MODE) CSettings::GetInstance().GetInt(CSettings::SETTING_VIDEOPLAYER_STEREOSCOPICPLAYBACKMODE);
   RENDER_STEREO_MODE mode = GetStereoMode();
 
+  g_infoManager.UpdateAVInfo();
```

Comment:
That may need some guidance.
g_graphicsContext.GetStereoMode() is used through application and renderer as the current 3D mode. The only place it gets set is from CStereoscopicsManager::ApplyStereoMode which comes from CStereoscopicsManager::OnPlaybackStarted though infomanager.

Can you explain how this information should flow?